src/Query/Result/ItemJournal.php

```diff
@@ -4,7 +4,7 @@
 
 use SMW\DIProperty;
 use SMW\DIWikiPage;
-use SMWDataItem as DataItem;
```

Comment:
Why was this changed?

Reply:
The alias makes the code ever so slightly less easy to understand.